local-proxy/package.json

```diff
@@ -17,7 +17,8 @@
     "@babel/cli": "^7.5.5",
     "@babel/core": "^7.5.5",
     "@babel/plugin-transform-modules-commonjs": "^7.5.0",
-    "@binaris/shift-db": "0.3.1",
```

Comment:
`@binaris/shift-db` as direct dependency of `@binaris/shift-local-proxy` is really fragile and probably incorrect, but perhaps it can be listed in `peerDependencies` (since it still works correctly but might print a warning message `"Could not require whitelisted module @binaris/shift-db"`, WDYT?

Reply:
Sounds reasonable enough.  Doing this.  I reckon eventually  we'll change the code only to emit warnings if a library is accessed.